[nri-bundle] Improve `nri-bundle` README (#846)
newrelic/helm-charts: charts/nri-bundle/README.md

# nri-bundle

![Version: 4.5.6](https://img.shields.io/badge/Version-4.5.6-informational?style=flat-square)

Groups together the individual charts for the New Relic Kubernetes solution for a more comfortable deployment.

**Homepage:** <https://github.com/newrelic/helm-charts>

## Bundled charts

> This chart do not deploy anything by itself but has many charts as dependencies install and upgrade the New Relic Kubernetes

Reviewer: Small grammar errors. My recommended fix:
"This chart [does not] deploy anything by itself but has many charts as dependencies. [This allows you to easily] install and upgrade the New Relic Kubernetes Integration using only one chart."

Author: English is not my first language so I really appreciate this feedback.

Thank you very much!

Author: Fixed in [#664266f](https://github.com/newrelic/helm-charts/pull/846/commits/664266f8b8372620475b4d84a853e68fe5f9ca13)

Integration using only one chart.

> In case you need more information of each component this chart installs or you are and advanced user that want to install each

Reviewer: "In case you need more information [about] each component this chart installs[,] or you are [an] advanced user that want to install each
component [separately], here is a list of components that this chart installs and where you can [find] more information about them:"

Author: Fixed in [#664266f](https://github.com/newrelic/helm-charts/pull/846/commits/664266f8b8372620475b4d84a853e68fe5f9ca13)

component separated, here is a list of components that this chart installs and where you can have more information about them:

| Component                    | Installed by default? | Description |
|------------------------------|-----------------------|-------------|

> | [newrelic-infrastructure](https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure) | Yes | This component will (in privileged mode) monitor the nodes and run integrations to monitor services inside the Kubernetes cluster (in any mode). It will also deploy a daemonset in the control plane nodes to give information about the health of the control plane and deployment to scrape the data from the KSM (see below) |

Reviewer 2: ```suggestion
| [newrelic-infrastructure](https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure) | Yes | Sends metrics about nodes, cluster objects (e.g. Deployments, Pods), and the control plane to New Relic. |
```

> | [nri-metadata-injection](https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection) | Yes | Webhook that will inject environments variables to all the pods so the APM agents have the context to know where they are running. |

Reviewer 2: ```suggestion
| [nri-metadata-injection](https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection) | Yes | Enriches New Relic-instrumented applications (APM) with Kubernetes information. |
```

> | [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics) | | kube-state-metrics (KSM) listens to the Kubernetes API server and generates metrics about the state of the objects. It is not focused on the health of the individual Kubernetes components, but on the health of the various objects inside, such as deployments and pods. |

Reviewer 2: ```suggestion
| [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics) | | Required for `newrelic-infrastructure` to gather cluster-level metrics. |
```

> | [nri-kube-events](https://github.com/newrelic/nri-kube-events/tree/main/charts/nri-kube-events) | | This component is an active watcher of event resource in the Kubernetes system that take those infrastructure events (like a pod has been created or a pod have just crashed) and pushes them to New Relic. |

Reviewer 2: ```suggestion
| [nri-kube-events](https://github.com/newrelic/nri-kube-events/tree/main/charts/nri-kube-events) | | Reports Kubernetes events to New Relic. |
```

> | [newrelic-infra-operator](https://github.com/newrelic/newrelic-infra-operator/tree/main/charts/newrelic-infra-operator) | | (Beta) This operator is meant to be used with Fargate clusters or serverless clusters that need to deploy the infrastructure agent as a sidecar per pod. |

Reviewer 2: ```suggestion
| [newrelic-infra-operator](https://github.com/newrelic/newrelic-infra-operator/tree/main/charts/newrelic-infra-operator) | | (Beta) Used with Fargate or serverless environments to inject `newrelic-infrastructure` as a sidecar instead of the usual DaemonSet. |
```

| [newrelic-k8s-metrics-adapter](https://github.com/newrelic/newrelic-k8s-metrics-adapter/tree/main/charts/newrelic-k8s-metrics-adapter) |  | (Beta) Provides a source of data for Horizontal Pod Autoscalers (HPA) based on a NRQL query from New Relic. |
| [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) |  | Sends logs for Kubernetes components and workloads running on the cluster to New Relic. |
| [nri-prometheus](https://github.com/newrelic/nri-prometheus/tree/main/charts/nri-prometheus) |  | Sends metrics from applications exposing Prometheus metrics to New Relic. |

> | [newrelic-pixie](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie) |  | Connects to the Pixie API and enables the New Relic plugin in Pixie. The plugin then sends data generated by a set of PxL scripts to New Relic. |

Reviewer: Maybe this focuses on the benefits of `newrelic-pixie` a little better? 
"Connects to the Pixie API and enables the New Relic plugin in Pixie. The plugin allows you to export data from Pixie to New Relic for long-term data retention."

Author: Fixed in [#664266f](https://github.com/newrelic/helm-charts/pull/846/commits/664266f8b8372620475b4d84a853e68fe5f9ca13)

> | [newrelic-k8s-metrics-adapter](https://github.com/newrelic/newrelic-k8s-metrics-adapter/tree/main/charts/newrelic-k8s-metrics-adapter) |  | (Beta) Provides a source of data for Horizontal Pod Autoscalers (HPA) based on a NRQL query from New Relic. |

Reviewer 2: I actually like this one!

> | [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) |  | Install a Daemonset on all the nodes that allow to send logs from the cluster to New Relic. |

Reviewer 2: ```suggestion
| [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) |  | Sends logs for Kubernetes components and workloads running on the cluster to New Relic. |
```

> | [nri-prometheus](https://github.com/newrelic/nri-prometheus/tree/main/charts/nri-prometheus) |  | Fetch metrics in the Prometheus metrics format and send them to the New Relic platform. |

Reviewer 2: ```suggestion
| [nri-prometheus](https://github.com/newrelic/nri-prometheus/tree/main/charts/nri-prometheus) |  | Sends metrics from applications exposing Prometheus metrics to New Relic. |
```

> | [Pixie](https://docs.pixielabs.ai/installing-pixie/install-schemes/helm/#3.-deploy) |  | Is an open source observability tool for Kubernetes applications that uses eBPF to automatically capture telemetry data without the need for manual instrumentation. |

Reviewer 2: Maybe someone from pixie wants to review these.
